src/views/Feed/Feed.js

```diff
@@ -0,0 +1,83 @@
+import _ from 'lodash'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import META from '../../utils/Meta'
+import { customPropTypes, getUnhandledProps } from '../../utils/propUtils'
+import * as sui from '../../utils/semanticUtils'
+import FeedContent from './FeedContent'
+import FeedDate from './FeedDate'
+import FeedEvent from './FeedEvent'
+import FeedExtra from './FeedExtra'
+import FeedLabel from './FeedLabel'
+import FeedLike from './FeedLike'
+import FeedMeta from './FeedMeta'
+import FeedSummary from './FeedSummary'
+import FeedUser from './FeedUser'
+
+function Feed(props) {
+  const { children, className, events, size } = props
+  const classes = cx('ui', className, size, 'feed')
+  const rest = getUnhandledProps(Feed, props)
+
+  if (!events) {
+    return <div {...rest} className={classes}>{children}</div>
+  }
+
+  const eventsJSX = events.map(({ key, ...eventData }, index) => {
+    const finalKey = key || index
```

Comment:
I think you mean `index` is an antipattern here.  In either case, 👍 to the proposed change.